tests/integration/telemetry/stats/prometheus/wasm/bad_wasm_filter_test.go

```diff
@@ -39,7 +40,14 @@ func TestBadWasmRemoteLoad(t *testing.T) {
 		Run(func(ctx framework.TestContext) {
 			// Test bad wasm remote load in only one cluster.
 			// There is no need to repeat the same testing logic in several different clusters.
-			cltInstance := common.GetClientInstances()[0]
+			var cltInstance echo.Instance
+			for _, i := range common.GetClientInstances() {
```

Comment:
could probably be: `common.GetClientInstances().GetOrFail(ctx, echo.InCluster(ctx.Clusters().Default()))`